imm_ukf_pda_track/nodes/imm_ukf_pda/imm_ukf_pda.cpp

```diff
@@ -54,8 +54,8 @@ ImmUkfPda::ImmUkfPda()
 
 void ImmUkfPda::run()
 {
-  pub_object_array_ = node_handle_.advertise<autoware_msgs::DetectedObjectArray>("/detection/objects", 1);
-  sub_detected_array_ = node_handle_.subscribe("/detection/fusion_tools/objects", 1, &ImmUkfPda::callback, this);
+  pub_object_array_ = node_handle_.advertise<autoware_msgs::DetectedObjectArray>("objects", 1);
+  sub_detected_array_ = node_handle_.subscribe("/detection/objects", 1, &ImmUkfPda::callback, this);
```

Comment:
Could you please make these something like "objects_in" and "objects_out" and then remap them in the launch files as necessary? Having one be relative and the other not is not ideal. Additionally, removing a middle-layer namespace is very problematic since the hard-coded namespace provided here can't be pushed into a new namespace and existing remappings will no longer work.

Reply:
That makes much more sense, thanks for pointing out!

Reply:
Thanks!